src/test/mir-opt/const-promotion-extern-static.rs

```diff
@@ -4,70 +4,12 @@ extern "C" {
 
 static Y: i32 = 42;
 
+// EMIT_MIR rustc.BAR.PromoteTemps.diff
+// EMIT_MIR rustc.BAR-promoted[0].ConstProp.after.mir
 static mut BAR: *const &i32 = [&Y].as_ptr();
```

Comment:
Why does a test with `extern-static` in its name test non-foreign `static`s?

Reply:
as a comparison. This test was created because a bug existed in extern statics but not in normal statics